bin/stacks/cron-stack.ts

```diff
@@ -94,6 +94,7 @@ export class CronStack extends cdk.NestedStack {
         REDSHIFT_DATABASE: RsDatabase,
         REDSHIFT_CLUSTER_IDENTIFIER: RsClusterIdentifier,
         REDSHIFT_SECRET_ARN: RedshiftCredSecretArn,
+        stage: stage,
```

Comment:
Ah I think this is because in your your last PR we moved stage out of envVars but never passed it directly to lambda env like this.